articles/devops-project/azure-devops-project-iot.md

```diff
@@ -0,0 +1,138 @@
+# Continuously deploy to IoT Edge applications with DevOps Projects
+
+Azure DevOps Projects presents a simplified experience where you can bring your existing code and Git repo or choose a sample application to create a continuous integration (CI) and continuous delivery (CD) pipeline to Azure.
+
+DevOps Projects also:
+
+- Automatically creates Azure resources, such as IoTHub

```

Comment:
should we say IoT Hub instead of IoTHub?

Reply:
Yeah correct, my bad!